AppKit/CPCompatibility.j

```diff
@@ -30,7 +30,8 @@ CPGeckoBrowserEngine                    = 1;
 CPInternetExplorerBrowserEngine         = 2;
 CPKHTMLBrowserEngine                    = 3;
 CPOperaBrowserEngine                    = 4;
-CPWebKitBrowserEngine                   = 5;
+CPWebKitBrowserEngine                   = 5;  // Safari
+CPBlinkBrowserEngine                    = 6;  // Chrome
```

Comment:
I've seen that and it's a good thing because internally we are not supposed to use engine detection. On the other hand we still have this CPBrowserIsEngine() public method and we don't want apps that use it to fail when they check the WebKit engine in Chrome.
The question behind that is : does Blink support all the features the Webkit engine supported at the time Blink was introduced ?. If yes (i suspect the answer is yes), then the current chrome  supports both engines. In this case, we should allow that by turning these constants into masks.